configs/go.json

```diff
@@ -137,39 +135,26 @@
     { "target": "libp2p/go-libp2p-kad-dht" },
     { "target": "libp2p/go-libp2p-kbucket" },
     { "target": "libp2p/go-libp2p-loggables" },
-    { "target": "libp2p/go-libp2p-mplex" },
-    { "target": "libp2p/go-libp2p-nat" },
-    { "target": "libp2p/go-libp2p-noise" },
     { "target": "libp2p/go-libp2p-peerstore" },
-    { "target": "libp2p/go-libp2p-pnet" },
     { "target": "libp2p/go-libp2p-pubsub-router" },
     { "target": "libp2p/go-libp2p-pubsub-tracer" },
-    { "target": "libp2p/go-libp2p-quic-transport" },
     { "target": "libp2p/go-libp2p-raft" },
     { "target": "libp2p/go-libp2p-record" },
     { "target": "libp2p/go-libp2p-relay-daemon" },
     { "target": "libp2p/go-libp2p-resource-manager" },
     { "target": "libp2p/go-libp2p-routing-helpers" },
-    { "target": "libp2p/go-libp2p-swarm" },
     { "target": "libp2p/go-libp2p-testing" },
-    { "target": "libp2p/go-libp2p-tls" },
-    { "target": "libp2p/go-libp2p-transport-upgrader" },
```

Comment:
Should we archive this one too?

https://github.com/libp2p/go-libp2p-transport-upgrader

Reply:
Looks like I forgot to deprecate this one!

Reply:
Will do so later today, but that shouldn’t stop us from merging this PR.